### DISCUSSION TITLE
Activating the cuda feature

### DISCUSSION THREAD

**Discussion by trago** [pykeio/ort#14] [Q&A] [answered]
Hi, as in other comments I must to say this is a great crate for using the onnruntime from rust. I have set it for a little project for inference and it just work well.

But, I'm not sure that I'm actually using cuda for the inference processing. I downloaded the onnxruntime with cuda compiled and use the ORT_STRATEGY=system and set the ORT_LIB_LOCATION, also I set the LD_LIBRARY_PATH so the onnxlibrary is loaded at runtime.

I check the cuda availability using `ExecutionProvider::cuda().is_available()` and it returns true so I use the cuda provider when creating the session. However I see the same performance as using CPU.

So my question is, how can I setup the _cuda_ feature as in the readme it says that to enable it we must to set this feature. Do you have an example of how to set this feature so I can test?

Thanks

**Comment by decahedron1** [comment, marked as answer]
Did you enable the `cuda` Cargo feature?

```toml
ort = { version = "1.14.0-alpha.0", features = [ "cuda" ] }
```

**Reply by trago** [reply]
That is what probably I'm doing wrong so my question is how to enable it. I added this to my cargo

```
ort = { git = "https://github.com/pykeio/ort", tag = "v1.14.0-alpha.0", fearures = ["cuda"] } # "1.14.0-alpha.0"
```

But I have this warning when running the cargo build
```
warning: /some/path/Cargo.toml: unused manifest key: dependencies.ort.fearures
```

So I might be doing it wrong. Do you have an example of how to enable `cuda`?

**Reply by decahedron1** [reply]
`features` is spelled wrong

**Reply by trago** [reply]
you are right, that is funny. Let me try again.

I'll let you know

**Reply by trago** [reply]
Hey!! Now it is using cuda.

I feel bad for this question hehe, but thanks for letting me see this error from me.

THanks!!
